README.md

```diff
@@ -1,3 +1,24 @@
-# dll_injector
+# Giuroll DLL Injector
 
 A simple injector for [giuroll](https://github.com/Giufinn/giuroll) and [giuroll-hagb](https://github.com/hagb/giuroll-hagb), modified from [dll injector by @kimjongbing](https://github.com/kimjongbing/dll_injector), not detected as virus by Windows Defender.
+It applies Giuroll to Hisoutensoku without requiring `SWRSToys`, and works akin to the legacy `SokurollLoader.exe` used in older copies of the game.
+
+## Compilation
+1. Navigate to this directory in the command prompt
+```
+cd <...>/giuroll/injector
+```
+
+2. Compile with
+```
+cargo +nightly-i686-pc-windows-msvc build --release
```

Comment:
The toolchain need not to be same with the target. It can simply use `nightly`, like
```
rustup install nightly
rustup +nightly component add rust-src --toolchain nightly
cargo +nightly build --target i686-win7-windows-msvc -Z build-std
```